Package.swift

```diff
@@ -59,6 +59,7 @@ let package = Package(
                 .copy("Foundation/GeoJSON/Fixtures/simple-line.geojson"),
                 .copy("Foundation/GeoJSON/Fixtures/polygon.geojson"),
                 .copy("Foundation/GeoJSON/Fixtures/multipolygon.geojson"),
+                .copy("Resources/MapInitOptionsTests.xib"),
```

Comment:
should this be a `.process` directive instead of a `.copy`?

Reply:
Nope. Not supposed to be public.

Reply:
?